tsconfig.json

```diff
@@ -0,0 +1,24 @@
+{
+  "compilerOptions": {
```

Comment:
Full list of [`tsconfig.json` settings](https://www.typescriptlang.org/tsconfig).